GenerateOppMessage.cmake

```
@@ -3,35 +3,29 @@ cmake_minimum_required(VERSION 3.1)
 include(CMakeParseArguments)
 
 # generate sources for messages via opp_msgc
+# Does NOT add to target include directories
 macro(generate_opp_message _msg_target)
     cmake_parse_arguments(_gen_opp_msg "" "" "MESSAGE_FILES" ${ARGN})
```

```suggestion
    cmake_parse_arguments(_gen_opp_msg "" "VERSION" "MESSAGE_FILES" ${ARGN})
    if(_gen_opp_msg_VERSION)
        set(_msg_version_arg "--msg${_gen_opp_msg_VERSION}")
    endif()
```
What could something like this be like?

---

you don't think it be like it is but it do